pkg/server/backgroundsvcs/background_services.go

```diff
@@ -34,7 +35,7 @@ func ProvideBackgroundServiceRegistry(
 	tracing tracing.Tracer, remoteCache *remotecache.RemoteCache, secretsService *secretsManager.SecretsService,
 	// Need to make sure these are initialized, is there a better place to put them?
 	_ *plugindashboards.Service, _ *dashboardsnapshots.Service, _ *pluginsettings.Service,
-	_ *alerting.AlertNotificationService, _ serviceaccounts.Service,
+	_ *alerting.AlertNotificationService, _ serviceaccounts.Service,  _ *resourceservices.ResourceServices,
```

Comment:
Since we import this in the `[http_server](https://github.com/grafana/grafana/pull/43951/files#diff-059f97ccda689ebea0e2f9981cd736d77b1fa01a719d95abae333ad40fb0b9c4R32)`, we don't need to have it here anymore.